src/components/MainPageContent.tsx

```diff
@@ -11,34 +11,34 @@ const axiosMainData = async (offset = 0, limit = 0) => {
   );
 };
 
-type contents = [] | { title: string; _id: string }[];
+export type Content = {
+  title: string;
+  _id: string;
+};
 
 function MainPageContent() {
+  const [contents, setContents] = useState<Content[]>([]);
+  const [isContentEmpty, setIsContentEmpty] = useState<boolean>(false);
+  const contentCount = useRef<number>(0);
   const observeRef = useObserver(() => {
-    addContent();
+    moreContent();
   });
-  const [contents, setContents] = useState<contents>([]);
-  const [noMoreContent, setNoMoreContent] = useState<boolean>(false);
-  const contentAddCount = useRef<number>(0);
-
-  const addContent = useCallback(async (limit = 6) => {
-    if (noMoreContent) return;
-    const { data } = await axiosMainData(
-      contentAddCount.current * limit,
-      limit
-    );
+  const moreContent = useCallback(async (limit = 9) => {
+    if (isContentEmpty) return;
+    const { data } = await axiosMainData(contentCount.current * limit, limit);
+    console.log(data);
     setContents((prevContents) => [...prevContents, ...data]);
-    contentAddCount.current++;
-    if (data.length !== limit) setNoMoreContent(true);
+    contentCount.current++;
+    if (data.length !== limit) setIsContentEmpty(true);
   }, []);
 
   return (
     <>
-      <Flex flexDir="column" pos="relative" p="15px 15px 50px 15px">
+      <Flex flexDir="column" pos="relative" p="15 15 50 15">
         <ContentList contents={contents} />
         <div ref={observeRef as React.MutableRefObject<HTMLDivElement>}></div>
-        <Box w="100%" p="20px" textAlign="center">
-          {noMoreContent ? (
+        <Box w="100%" p="20" textAlign="center">
```

Comment:
```suggestion
        <Box w="100%" p={20} textAlign="center">
```

20으로 하신 의도가 20px이신 것 같은데 20과 20px은 다른 단위입니다.
<img width="527" alt="image" src="https://github.com/prgrms-fe-devcourse/FEDC4_Campers_Jaeho/assets/66409882/f1995bfa-6f4b-447f-8237-38b1e09b829a">
공식문서를 다시 한 번 확인해보시는게 좋을 것 같아요